views/home/references.ejs

```diff
@@ -35,7 +35,8 @@
         <td>
             <input type="checkbox" ng-change="approve(reference.id, reference.approved)"
                    ng-model="reference.approved" ng-if="user.isMod"/>
-            <span ng-if="user.isMod && refUser && reference.edited">*</span>
+            <span ng-if="user.isMod && refUser && reference.edited"><font color="red" title="This reference was edited at some point.">*</font></span>
```

Comment:
Very good. Only thing, please add a class for this and set the colour in css, not hard code it here. But apart from that, the title was a very good idea.